configlib-core/src/test/java/de/exlll/configlib/ExampleConfigurationTests.java

```diff
@@ -30,7 +30,7 @@ class ExampleConfigurationTests {
             .build();
 
     private final FileSystem fs = Jimfs.newFileSystem();
-    private final Path yamlFile = fs.getPath("/tmp/config.yml");
+    private final Path yamlFile = fs.getPath(TestUtils.createPlatformSpecificFilePath("/tmp/config.yml"));
```

Comment:
I think `createPlatformSpecificFilePath` should be statically imported in all files that use it. What do you say?

Reply:
Yeah, you're right.